cmd/helm/status.go

```diff
@@ -68,6 +68,7 @@ func (s *statusCmd) run() error {
 
 	fmt.Fprintf(s.out, "Last Deployed: %s\n", timeconv.String(res.Info.LastDeployed))
 	fmt.Fprintf(s.out, "Status: %s\n", res.Info.Status.Code)
+	fmt.Fprintf(s.out, "Resources:\n%s\n", res.Info.Status.Resources)
```

Comment:
You can use the uitable (https://github.com/gosuri/uitable) pkg to format this output. See example here: https://github.com/kubernetes/helm/blob/master/cmd/helm/repo.go#L92